.github/labeler.yml

```diff
@@ -0,0 +1,9 @@
+test:
```

Comment:
I was thinking of something more self-explanatory
`with tests` or `contains tests`